installer/bundle/bundle_skel.sh

```diff
@@ -1550,12 +1550,12 @@ you should first purge the existing installation and then install using the --sk
         fi
 
         # Update OMS Agent
-        shouldInstall_omsagent
         rm -f "$OMS_CONSISTENCY_INVOKER" > /dev/null 2>&1
```

Comment:
Sorry, one more thought ... is it safe to always rm the OMS_CONSISTENCY_INVOKER cron job here? Can't tell what scenarios it will be recreated per https://github.com/microsoft/OMS-Agent-for-Linux/blob/c7cff7009a78254c89b3e6d0ea24302e3b7930ae/installer/bundle/bundle_skel.sh#L1619

Reply:
@hestolz  yes, cron job needs to be deleted before agent and dsc are upgraded because if it stays in running it might pull config while these packages are being upgraded. It is being created by DSC LCM engine that is why you do not see it here. There are better ways of doing cron job clean up, but changing it now would not bring any functional impact.